.github/workflows/docker-hub.yml

```diff
@@ -24,12 +24,13 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: lowercase rundeck version
+        run: |
+          echo "RUNDECK_TAG=$(echo ${{ matrix.rundeck_version }} | tr [:upper:] [:lower:])" >>${GITHUB_ENV}
       - name: Build and export to Docker
         uses: docker/build-push-action@v2
         with:
-          context: .
-          build-args:
-            - RUNDECK_VERSION=${{ matrix.rundeck_version }}
+          build-args: "RUNDECK_VERSION=${{ matrix.rundeck_version }}"
           file: Dockerfile
           load: true
-          tags: fretlink/rundeckforci-${{ matrix.rundeck_version }}-latest
+          tags: fretlink/rundeckforci-${{ env.RUNDECK_TAG }}-latest
```

Comment:
Is it normal that there’s a mix between RUNDECK_TAG and RUNDECK_VERSION?

Reply:
Yes because tag has to be lowercase but in fact I'm doing it wrong